pkg/controller/volume/cache/desired_state_of_world_test.go

```diff
@@ -987,10 +991,10 @@ func Test_GetVolumesToAttach_Positive_TwoNodesThreeVolumes(t *testing.T) {
 			generatedVolume2Name1,
 			generatedVolume2Name2)
 	}
-	pod3Name := types.UniquePodName("pod3-uid")
+	pod3Name := "pod3-uid"
 	volume3Name := api.UniqueVolumeName("volume3-name")
 	volume3Spec := controllervolumetesting.GetTestVolumeSpec(string(volume3Name), volume3Name)
-	generatedVolume3Name, podAddErr := dsw.AddPod(pod3Name, volume3Spec, node1Name)
+	generatedVolume3Name, podAddErr := dsw.AddPod(controllervolumetesting.NewPod(pod3Name, pod3Name), volume3Spec, node1Name)
 	if podAddErr != nil {
 		t.Fatalf(
 			"AddPod failed for pod %q. Expected: <no error> Actual: <%v>",
```

Comment:
what kind of tests you want to add? NewPod() is a utility function
in controllervolumetesting which returns a pod object with given pod uid
and name.

On Fri, Jun 17, 2016 at 5:59 PM, Saad Ali <notifications@github.com> wrote:

> In pkg/controller/volume/cache/desired_state_of_world_test.go
> <https://github.com/kubernetes/kubernetes/pull/27576#discussion_r67592057>
> :
>
> >  	volume3Name := api.UniqueVolumeName("volume3-name")
> >  	volume3Spec := controllervolumetesting.GetTestVolumeSpec(string(volume3Name), volume3Name)
> > -	generatedVolume3Name, podAddErr := dsw.AddPod(pod3Name, volume3Spec, node1Name)
> > +	generatedVolume3Name, podAddErr := dsw.AddPod(controllervolumetesting.NewPod(pod3Name, pod3Name), volume3Spec, node1Name)
> >  	if podAddErr != nil {
> >  		t.Fatalf(
> >  			"AddPod failed for pod %q. Expected: <no error> Actual: <%v>",
>
> If you have time, please add new tests explicitly for the NewPod() method
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27576/files/14b7176331c4564bf448202808db59198a034cab#r67592057>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASSNxTdfdA3mcRv4RYk6AIPSLp9Llt49ks5qM0LZgaJpZM4I350f>
> .
>



-- 
- Jing